python-opendata/opendata/download.py

```diff
@@ -0,0 +1,85 @@
+from collections import namedtuple
+import glob
+from os import listdir, path, remove
+from shutil import rmtree
+import zipfile
+
+from osfclient import api, cli
+import pandas as pd
+
+
+PROJECT_ID = '6hfpz'
+
+
+Args = namedtuple(
+    'Args',
+    ['project', 'remote', 'local', 'username', 'password', 'force', 'output'],
+)
+
+
+class OpenDataClient:
+    def __init__(self, project_id=PROJECT_ID):
+        self.args = Args(
+            project=project_id,
+            remote=None,
+            local=None,
+            username=None,
+            password=None,
+            force=False,
+            output=None,
+        )
+        self.osf = api.OSF()
+        self.project = self.osf.project(project_id)
+        self.store = next(self.project.storages)
+
+    def download_athlete(self, athlete_id):
+        for f in self.store.files:
```

Comment:
```python
map_file = {f.name.rstrip('.zip'): f for f in self.store.files}

for id in athlete_id:
    osf_file = map_file[id]
    file_path = path.join('.opendata_storage/', osf_file.name)
    args = self.args._replace(remote=osf_file.path, local=file_path)
    cli.fetch(args)
    elf._unzip_and_delete_file(file_path)
```

Reply:
@AartGoossens Or the break it to actually avoid iterating for all files?

Reply:
Yes that was the idea. I realize this code is very rough and needs some attention (and optimization). Do you agree to leave it like this for now? We can create a new PR to improve these things.

Reply:
I am fine for a start. You probably need to start experimenting some stuff right away so until the first release everything can get crazy :). I think that there is need for documentation and test anyway. I would be happy to contribute on those one (it could help me to catch up with what is happening)